### ISSUE TITLE
All plugins re-installation is broken due bad ID name in plugin.xml file

### ISSUE THREAD

**Issue by phyr0s** [HMS-Core/hms-cordova-plugin#173]
**Description**

When install plugin and remove the platforms/plugins folder and want to reinstall from package.json the reinstall fails

The problem is because npm search the @hmscore/cordova-plugin-hms-availability and the name of the plugin id is cordova-plugin-hms-availability so when you install in the package.json in the cordova plugin object it added as cordova-plugin-hms-availability

Force change @hmscore/cordova-plugin-hms-availability locally it didn't fix the issue because next reinstall cordova-plugin-hms-availability will appear again in the cordova.plugins object in the package json.

Basically there is not cordova-plugin-hms-availability package in NPM repository and fails because the name of repo is @hmscore/cordova-plugin-hms-availability

Happen with all plugins, all version of cordova, ionic etc....
